springwolf-plugins/springwolf-kafka-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/ClassLevelKafkaListenerScanner.java

```diff
@@ -153,7 +153,11 @@ private Set<Method> getAnnotatedMethods(Class<?> component) {
     }
 
     private ChannelItem buildChannel(Set<Method> methods, Map<String, ? extends OperationBinding> operationBinding) {
+        String operationId = methods.stream().findFirst().map(it->it.getName() + "_subscribe").orElse("");
```

Comment:
I believe this should be `_publish`, right?

Reply:
True, I will update the PR.
Following the discussion in discord, we should consider moving the scanners to `publish` and `subscribe` packages. I also find this highly confusing.
What do you think? In any case, this refactor would be outside of this PR.

Reply:
I agree, there should be separate packages.